docs/develop/01_sdk/02_cookbook/05_messaging/01_messaging.md

```diff
@@ -0,0 +1,57 @@
+---
+id: messaging_book
+title: Generate a Message
+---
+
+import TsJsBlock from '@site/src/components/TsJsBlock';
+
+import GenerateMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/01_generate_message.ts';
+import EncryptMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/02_encrypt_message.ts';
+import DecryptMessage from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/messaging/03_decrypt_message.ts';
+
+KILT defines a [unicast](https://en.wikipedia.org/wiki/Unicast) [messaging protocol](../../../../concepts/06_messaging.md).
```

Comment:
We are in the concepts section, I think this is too much to start the whole paragraph with.

Reply:
Maybe, this is the disconnect people are talking about. It doesn't have information that helps form the reader. We jump into code.

This can be a problem if we just link to the concepts they completely move away from the code.